tests/tests.py

```diff
@@ -1,4 +1,4 @@
-from datetime import datetime
+from datetime import datetime, 
```

Comment:
`datetime,` ?

Reply:
already fixed. yeah, I am watching travis logs too and trying to pass 😄